.gitattributes

```diff
@@ -52,20 +52,22 @@ README*                  ocaml-typo=missing-header
 /tools/ci/appveyor/appveyor_build.cmd      ocaml-typo=long-line,very-long-line,missing-header,non-ascii text eol=crlf
 /tools/ci/appveyor/appveyor_build.sh       ocaml-typo=non-ascii
 
+# depend files are auto-generated
+.depend                  ocaml-typo=missing-header
```

Comment:
This is supposed to be an automatic rule, but I think they're only invoked when check-typo is running in it's original "entire repo" mode. I think it would be better to fix that, than paper over it here - or at least it should be that the comment notes that the line is only required because of a bug in check-typo.

Reply:
Thanks, I will investigate.